packages/opencensus-core/src/trace/instrumentation/types.ts

```diff
@@ -39,3 +41,18 @@ export interface Plugin {
 export type PluginNames = {
   [pluginName: string]: string;
 };
+
+/**
+ * Each key should be the name of the module to trace, and its value
+ * a mapping of a property name to a internal plugin file name.
+ */
+export type PluginInternalFiles = {
+  [versions: string]: PluginNames;
+};
+
+/**
+ * Maps a name (key) representing a internal file module and its exports
+ */
+export type ModuleExports = {
```

Comment:
The type name here should be more descriptive. Something like `ModuleExportsMapping` or something like that. But I wonder if you really need this type to be defined explicitly, because it's so broad. (Why not just define it locally in `base-plugin.ts`.)

Reply:
Agree. Done !